Mesh_3/include/CGAL/Mesh_3/Refine_facets_manifold_base.h

```diff
@@ -239,7 +242,20 @@ class Refine_facets_manifold_base
               << biggest_sq_dist << std::endl;
 #endif // CGAL_MESHES_DEBUG_REFINEMENT_POINTS
 
-    return biggest_facet;
+    return { biggest_facet, biggest_sq_dist };
+  }
+
+  bool is_smaller_than_min_size(const FT& sq_dist) const
+  {
+    if(!m_facet_min_size)
+      return false;
+    return sq_dist < CGAL::square(*m_facet_min_size);
```

Comment:
As this is the only usage of the value of `m_facet_min_size`, I suggest it is renamed `m_squared_facet_min_size` and constructed directly as a square.